### DISCUSSION TITLE
ArgoCD to deploy a upstream(bitnami) chart. Values.yml do not apply config

### DISCUSSION THREAD

**Discussion by hamps-contrib** [argoproj/argo-cd#5881] [Q&A] [answered]
Hi,
I am using the https://github.com/argoproj/argocd-example-apps/tree/master/helm-dependency , to deploy bitnami charts in my kubernetes cluster. I am doing the same as the example linked above and it works in Argocd, it gets deployed and it finds the values.yml file. How ever, the configuration is not applied in the helm chart deployment. I still get basic config, even though argocd "parameters" UI shows my specified configuration from my values.yml

I am running Argocd 2.0 and bitnami/kube-prometheus chart.

This is my private git repo structure:
└── kube-prometheus (Directory)
     ├── Chart.yaml 
     ├── README.md
    └── values.yaml <-- This config is not used for some reason?

dependencies:
  - name: kube-prometheus
    version: 4.2.0
    repository: https://charts.bitnami.com/bitnami

**Comment by jannfis** [comment, marked as answer]
Without seeing your `values.yaml` I can just guess here that you might be using a verbatim version of `values.yaml` for the `kube-prometheus` stack. 

When Helm is rendering a dependency, it will require the dependencies' values to be provided below the dependency's name as key, so that Helm will know which part of the `values.yaml` to pass to the dependency when rendering.

So, in order to provide the values to a dependency named `kube-prometheus` and given your original `values.yaml` be like

```yaml
foo: bar
bar:
  foo: bar
```
it should be stored in your `values.yaml` like the following instead:

```yaml
kube-prometheus:
  foo: bar
  bar:
    foo: bar
```

HTH.

**Reply by hamps-contrib** [reply]
Thanks, Indeed that was the misstake i did. 

It works now.
